modules/sandboxed-containers-common-terms.adoc

```diff
@@ -0,0 +1,34 @@
+//Module included in the following assemblies:
+//
+// * sandboxed_containers/understanding_sandboxed_containers.adoc
+
+[id="sandboxed-containers-common-terms_{context}"]
+
+= Common terms
+
+.Sandbox
+A sandbox is a controlled environment where programs can run. Within the controlled environment, you can execute untested or untrusted programs without risking harm to the host machine or the operating system. A sandbox keeps your workloads isolated from the rest of the system.
+
+In the case of {sandboxed-containers-first}, sandboxing is achieved by running workloads in a different kernel using virtualization, providing enhanced control over the interactions between multiple workloads running on the same host.
+
+.Pod
+A pod is a kubernetes and OpenShift construct that represents resources where containers can be deployed. In other words, containers run inside pods. Pods are used for example to specify resources that can be shared between multiple containers.
+
+In the case of {sandboxed-containers-first}, a pod is implemented as a virtual machine. Several containers can run in the same pod, and therefore in the same virtual machine.
+
+.OpenShift sandboxed containers Operator
```

Comment:
+1 - you can (and should) use the attributes everywhere but in the `_topic_map.yml` file. I see a couple of OpenShift sandboxed containers and RHCOS references in headings that should be swapped to attributes.

Reply:
@maxwelldb + @kalexand-rh I was leaving it off from headings because of PV2 compliance. PV2 checklist: Do not use {variables} in title/chapter names or IDs

Reply:
Yes, ignore the PV2 checklist for now. FWIW, that guidance is only temporary.